Week2/MohammadDarwesh/mohammaddarwesh.js

```diff
@@ -0,0 +1,180 @@
+// for the first task i will do it with many ways just for practising.
+console.log ("Halo, dunia! ,Ciao, mondo! ,Hola, mundo!");
+
+let indonesian = "Halo, dunia!";
+let italian = "Ciao, mondo!";
+let spanish = "Hola, mundo!";
+console.log(indonesian ,italian, spanish );
+
+let helloWorld = [`${indonesian}` ,`${italian}` ,`${spanish}`];
+console.log(helloWorld);
+console.log(helloWorld[0]);
+console.log(helloWorld[1]);
+console.log(helloWorld[2]);
+
+let helloWorld2 = ["Halo, dunia!" ,"Ciao, mondo!" ,"Hola, mundo!"];
+console.log(helloWorld2);
+console.log(helloWorld2[0]);
+console.log(helloWorld2[1]);
+console.log(helloWorld2[2]);
+console.log(helloWorld2.join("-"));
+console.log(helloWorld2.reverse());
+
+let helloWorld3 = {
+    indonesian: "Halo, dunia!",
+    italian: "Ciao, mondo!",
+    spanish: "Hola, mundo!"
+   };
+console.log(helloWorld3);
+console.log(helloWorld3.indonesian);
+console.log(helloWorld3.italian);
+console.log(helloWorld3.spanish);
+// end of task 1.
+
+
+// task 2:
+console.log("I'm awesome");
+
+// task 3:
+let x;
+console.log("the value of my variable x will be: 32");
+console.log(x);
+x = 32;
+console.log("x will be :32 ");
+console.log(x);
+
+// task 4:
+let y = "Hello, Yash"
+console.log("the value of my string will be: Hello, Yash ");
+console.log(y);
+y = "Hiiii, Yash"
+console.log("the value of my string will be: Hiiii, Yash ");
+console.log(y);
+
+// task 5:
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+console.log(a > z , z > a );
+if (z > a) {
```

Comment:
Aah. There is a bug here which you didn't notice because the code never reached the `else` block. You will notice it if you maybe did something like: `if (z < a)......`

Anyways, the bug is because of how scoping of variables work when created using `let` or `const` or `var` keyword. We will be covering that in our tomorrow's class. 🙂 

Other than that, you could have used `const` keyword for creating variables instead of `let` for `z` and `a`.

Reply:
Also, you could also use the `Math.max` method here instead of writing the `if-else` block. Please find some more information about this method here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max

Reply:
thank you for explain it to me in the class .....and i am start using strict mode to avoid this kinds of bug.
about the Math.max ......if the purpose from the question to use  Math.max i think also will be better if the question more specific .